### DISCUSSION TITLE
11.12.23

### DISCUSSION THREAD

**Discussion by Roadraudi** [Roadraudi/Codespace_Repository#6] [Announcements]
In dieser Vorlesung haben wir uns mit unterschiedlichen Programmiersprachen und der jeweiligen Energie Effizienz auseinander gesetzt. Dies bezüglich haben wir uns die Statistiken:  "Proceedings of the 10th ACM SIGPLAN International Conference on Software Language Engineering (SLE 2017)" herangezogen.
Dabei stellte sich heraus das Python eine sehr Energie Effiziente Sprache ist im Gegenteil zu z.B. Java.

Meine Python Code Übungen waren bis zum heutigen Datum:
-Lebenslauf Generator
-Einkaufsliste
-Algorithmus von Euklid
-Primzahlen erkennen
-Glücksspiel auf Basis von erraten von Zufallszahlen
-Gehaltsrechner

**Comment by salmuuki** [comment]
Maschine

**Comment by SirPatschiii** [comment]
Interessante Ideen die hier umgesetzt wurden, insbesondere auch die Herangehensweise.
